ansible/utils/install_lxc_packages.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+- hosts: localhost
+  become: true
+  vars:
+    ansible_python_interpreter: /usr/bin/python3
+  tasks:
+    - name: Install lxc packages
+      package:
+        update_cache: yes
+        name:
+          - "lxc"
+          - "python3-lxc"
+          - "lxc-templates"
+        state: latest
```

Comment:
don't set it to `latest`, we don't want to upgrade the existing ones.
set it to `present` only.

Reply:
changed latest to present